qubesadmin/tools/qvm_prefs.py

```diff
@@ -141,9 +141,12 @@ def process_actions(parser, args, target):
         return 0
 
     try:
-        value = getattr(target, args.property)
-        if value is not None:
-            print(str(value))
+        if target.property_is_default(args.property) and args.hide_default:
```

Comment:
Better swap order, to save one admin api call in the common case.

Reply:
Done